evoting/service/state_test.go

```diff
@@ -0,0 +1,52 @@
+package service
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNonce(t *testing.T) {
+	n1, n2, n3 := nonce(10), nonce(10), nonce(10)
+	assert.Equal(t, 10, len(n1), len(n2), len(n3))
+	assert.NotEqual(t, n1, n2, n3)
```

Comment:
Same here - the test should be on three lines (n1, n2 - n2, n3 - n3, n1)

Reply:
Fixed.